src/containers/tutor-profile-page/video-presentation/VideoPresentation.jsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from 'react-i18next'
+import { Box, Typography } from '@mui/material'
+import VideoBox from '~/components/video-box/VideoBox'
+import videoImg from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
+
+
+const VideoPresentation = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box className='section' data-testid='videoPresentation' sx={ { flexDirection: 'column' } }>

```

Comment:
you don`t use this data-testid, so you can remove it

Reply:
fixed